tonilopezmr/CaptchaPartOne.kt

```diff
@@ -0,0 +1,46 @@
+package day1
+
+import org.junit.Test
+import org.junit.Assert.*
+
+
+
+class CaptchaPartOne {
+
+    fun sequence(sequence: String) : Int {
+        val captcha = (sequence + sequence[0])
```

Comment:
Uhh I like this better than:

if sequence[0] == sequence[-1]:
    acc += sequance[0]

:+1:

Reply:
What? do you prefer a if? that simple sum?

Reply:
Nono I say I like it **better** 😄